src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -85,6 +85,11 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             // An exception here will crash VS
             try
             {
+                if (!args.IsSolutionOpen)
+                {
+                    this.configurationProvider.WriteConfiguration(BindingConfiguration.Standalone);
```

Comment:
The binding state for new connected mode is now tracked in memory, so if we were in new connected mode we need to reset the state to standalone when the solution closes.
Previously this wasn't necessary as we detected the binding state by looking for files in the open solution.